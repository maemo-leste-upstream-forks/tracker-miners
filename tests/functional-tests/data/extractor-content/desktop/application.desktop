[Desktop Entry]
Name=Foo
Comment=my awesome application
Type=Application
Icon=foo
Exec=/usr/bin/foo
