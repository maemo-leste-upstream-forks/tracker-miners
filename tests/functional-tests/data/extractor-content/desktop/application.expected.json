{
    "test": {
        "Filename": "application.desktop"
    },
    "metadata": {
        "@graph": [
	    {
		"@type": [
		    "nfo:SoftwareApplication",
		    "nfo:Executable"
		],
		"nfo:softwareCmdLine": "/usr/bin/foo",
		"nfo:softwareIcon": {
		    "@id": "urn:theme-icon:foo",
		    "@type": [
			"nfo:Image"
		    ]
		},
		"nie:comment": "my awesome application",
		"nie:title" : "Foo"
	    }
	]
    }
}
